posthog/urls.py

```diff
@@ -180,11 +180,14 @@ def logout(request):
 ]
 
 if settings.DEBUG:
+    import debug_toolbar # type: ignore
```

Comment:
This should probably have the same try/except pattern in case someone uses DEBUG to debug live and hasn't installed debug_toolbarr

Reply:
sounds safe